.github/workflows/python-validate.yml

```diff
@@ -129,3 +131,14 @@ jobs:
 
           python3 -m pip install poethepoet pytest
           python3 -m poethepoet test
+
+      - name: Run AWS Lambda SDK unit tests
+        if: steps.pathChanges.outputs.lambdaSdkSchema == 'true'
+        run: |
+          cd python/packages/sdk
+          python3 -m pip install .
+          cd -
```

Comment:
I think this doesn't belong here.

This CI setup, I assume should introduce `Install AWS Lambda SDK project and dependencies` (which installs `aws-lambda-sdk` dependencies, also in this case ideally `sdk` should be installed from pip and not referenced  from side folder - it's how it works in Node.sj) and  `Run AWS Lambda SDK unit tests` which runs tests in it

Reply:
I agree, I was under the impression that steps ran independently and didn't share state, but that's not the case.